### ISSUE TITLE
Pips on cards are not centred

### ISSUE THREAD

**Issue by inspirobot-npross** [inspirobot/jester#2]
Need someone who knows CSS to make the pips on the card all render from centers.  The current positions seem to be aligned to the left of the glyph and the glyphs all have different amounts of padding so different suits appear at different positions...

**Comment by simonthomson** [comment]
I propose switching to the PNGs I created. They are all exactly 256 by 256 pixels.

**Comment by jeffro426** [comment]
PNG images will not scale gracefully. Perhaps we could create perfectly square SVG files instead?

**Comment by simonthomson** [comment]
Yes, I'm good with that. I don't really know how to create svgs properly though

**Comment by simonthomson** [comment]
If you're interested Jeff, you could create some svgs for the suit symbols. These were the ones I came up with, but they are open to modification/discussion: 
![cloud_symbol_256](https://user-images.githubusercontent.com/54864020/148126570-5ed946c8-3cde-460e-872b-279db2c02675.png)
![club_symbol_256](https://user-images.githubusercontent.com/54864020/148126572-bf15e116-efc7-41a7-9806-6cf37df92621.png)
![cross_symbol_256](https://user-images.githubusercontent.com/54864020/148126574-b0b5b3ee-7b96-4f45-81e9-0c9508b445e0.png)
![diamond_symbol_256](https://user-images.githubusercontent.com/54864020/148126576-bcaa3601-c974-4f33-92f6-31691116fe1e.png)
![flower_symbol_256](https://user-images.githubusercontent.com/54864020/148126577-61a2ce49-1e5e-4270-96ca-f7b6286bb570.png)
![heart_symbol_256](https://user-images.githubusercontent.com/54864020/148126578-196f9d74-2dcd-46cb-a465-ffea02664836.png)
![moon_symbol_256](https://user-images.githubusercontent.com/54864020/148126579-2b99cec1-fb6b-45d5-b893-74a8bfeecd95.png)
![spade_symbol_256](https://user-images.githubusercontent.com/54864020/148126581-f2b477ab-eaee-4757-8a30-f2c2d4bc35ed.png)
![star_symbol_256](https://user-images.githubusercontent.com/54864020/148126583-827ebc50-c386-4982-88ef-38e56d552459.png)
![tree_symbol_256](https://user-images.githubusercontent.com/54864020/148126584-7c9a8f4a-19d3-4dc1-951e-1a88e99c5378.png)

**Comment by jeffro426** [comment]
Happy to help! Yes, I can export these from Illustrator in SVG format and attach them to this thread.

**Comment by jeffro426** [comment]
I will make sure they are perfectly square. I will use 256px by 256px artboards unless you'd prefer a different size.

**Comment by jeffro426** [comment]
Please take a look at these SVG icons and let me know if they look OK. These are scaled to a square resolution 256px by 256px.
[Archive.zip](https://github.com/inspirobot/jester/files/7815804/Archive.zip)

**Comment by inspirobot-npross** [comment]
@jeffro426 Is there a way they can be converted to vectors?  I think these are just pngs in svg clothing. from inside the svg :(xlink:href="data:image/png;base64,)

**Comment by jeffro426** [comment]
Definitely! Yes, I can convert the graphics to vector first and export new SVG files.

**Comment by jeffro426** [comment]
I converted the artwork to vector and saved them in square SVG files
[Suits.zip](https://github.com/inspirobot/jester/files/7829896/Suits.zip)

**Comment by inspirobot-npross** [comment]
@jeffro426  ok i know I'm getting picky but I uned to work in a company whose sole reason for existing was to make graphics and text look good on television for 2 decades...  
When these get drawn by the browser at small sizes the anti-aliasing looks terrible.  
I'm not sure why.  Maybe the browser just sucks at antialiasing at small sizes.  Font renders have all kinds of hinting and are optimized for drawing vectors at small sizes... so maybe there is no fixing it.
Some of the svgs seem extra complicated (ie likely more lines and curves than actually necessary). 
Is there an option in illustrator to simplify curves and to make sure that it is using curves instead of a lot of line segments?
I have no idea if this will improve the rendering or not...

Of course this is not a showstopper or anything... I am going to see how hard it is to make the cards render with these svgs and if it really drives me crazy in the future there is always the option to just render all the cards as pixel-perfect images...

**Comment by jeffro426** [comment]
That's an interesting observation @inspirobot-npross !

I'd be glad to fix these vector images. Of course they look fine in my system  .

Yes, I can remove any extra geometry. I let Illustrator convert the GIF to vector and deleted the white background tiles. 

Do any of the icons scale properly or do they all show anti-aliasing?

I ask because some appeared to be vector based without background tiles (e.g., the spade, heart, club, and diamond suits).

**Comment by inspirobot-npross** [comment]
@jeffro426 I already manually removed the backgrounds. I'm not sure it made much difference.  I will experiment some more.

**Comment by jeffro426** [comment]
I tried a smaller scale version of the Suit icons at 52x52px which won't need to be rescaled as aggressively. Perhaps that will look cleaner. 

I double checked for any extra geometry and the objects are clean and simple. The Blue Cloud had an artifact which I removed which likely wasn't visible in the Browser.

[Suits_52x52.zip](https://github.com/inspirobot/jester/files/7840507/Suits_52x52.zip)
